web/impact/impact/views/algolia_api_key_view.py

```diff
@@ -69,16 +69,16 @@ def get(self, request, format=None):
         public_key = _get_public_key(params, search_key)
         return Response({
             'token': public_key,
-            'index_prefix': settings.ALGOLIA_INDEX_PREFIX,
+            'index_prefix': settings.ALGOLIA['INDEX_PREFIX'],
```

Comment:
I see how these new dict-based settings are handled in `opsworks_settings.py.in`, but I can't see how it works locally. Can you indulge me with an explanation?

Reply:
Locally this also [works the same way](https://github.com/masschallenge/accelerate/pull/2660/files#diff-b5047624c5074e382f8be3d058c1b6b2R355)

All thats happening here is that impact previously had its own algolia settings (ALGOLIA_INDEX_PREFIX, ALGOLIA_INDEX_PREFIX, STAFF_SEARCH_ONLY_API_KEY etc) - we are just porting this over to use the settings we already have for this in accelerate.

Reply:
Aha, I confused myself by accidentally looking at at impact-api's settings.py instead of accelerate's.